.github/workflows/publish.yml

```diff
@@ -4,6 +4,9 @@ on:
   release:
     types:
       - published
+  push:
+    branches:
+      - "bug/fix-publish-pipeline"
```

Comment:
Should we remove this? 😄

Reply:
Actually, I already pushed a commit to remove it 🙈 but not sure why it's not here
<img width="991" alt="Screenshot 2566-07-05 at 09 35 28" src="https://github.com/nimblehq/stylelint-config-nimble/assets/1772999/17745236-7953-4780-9954-c32f568d9b06">

Reply:
Fixed by push an extra commit 😂